criu/fdstore.c

```diff
@@ -53,17 +52,8 @@ int fdstore_init(void)
 		return -1;
 	}
 
-	if (!opts.unprivileged || has_cap_net_admin(opts.cap_eff)) {
```

Comment:
pls, add a few lines in the commit message to explain what problem you are solving

Reply:
Thanks, added the following:

>SO_SNDBUFFORCE/SO_RCVBUFFORCE require root or CAP_NET_ADMIN.
We can use SO_SNDBUF/SO_RCVBUF in some cases and avoid
needing elevated privileges.